packages/framework/src/Resources/views/Admin/Content/PersonalData/index.html.twig

```diff
@@ -5,71 +5,80 @@
 
 {% block main_content %}
     {{ render(controller('ShopsysFrameworkBundle:Admin/Domain:domainTabs')) }}
-        {{ form_start(form) }}
-    <div class="wrap-divider">
-        <div class="form-group">
+    {{ form_start(form) }}
+        <div class="wrap-divider">
             <div class="form-group margin-bottom-20">
                 <h2 class="form-group__title">
                     {{ 'Link to the personal information overview'|trans }}
                 </h2>
-                <div class="form-line">
-                    <div class="form-line__line">
-                        <div class="form-line__label">
-                            {{ getDomainName(domainId) }}
-                            {% if isMultidomain() %}
+                <div>
```

Comment:
Why is this `div` element important? Can't we just get rid of it?

Reply:
It makes inside element form-line first and last child - and it means that gets border radius

Reply:
That makes sense, thanks!